Model/Customer/Message/VerifyCustomerByTokenMessage.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\SyliusConsumerBundle\Model\Customer\Message;
+
+class VerifyCustomerByTokenMessage
```

Comment:
there is no other possible way to verify it `VerifyCustomerMessage` should be enough